spec/calendar.html

```diff
@@ -167,7 +167,6 @@ <h1>CalendarMonth ( _calendar_, _dateLike_ )</h1>
       <emu-alg>
         1. Assert: Type(_calendar_) is Object.
         1. Let _result_ be ? Invoke(_calendar_, *"month"*, « _dateLike_ »).
-        1. If _result_ is *undefined*, throw a *RangeError* exception.
```

Comment:
ToPositiveInteger won’t throw if the result is undefined - why remove this check?

Reply:
The commit message for that change has a more detailed explanation. Basically it's:
- `ToNumber(undefined)` -> NaN
- `ToIntegerOrInfinity(NaN)` -> 0
- `ToPositiveInteger(0)` -> throws a RangeError

Reply:
ah, i wouldn’t expect ToPositiveInteger to throw on zero.

Reply:
I agree, and this is in line with my suggestions at https://github.com/tc39/proposal-temporal/issues/2112#issuecomment-1148678997 .
> * **ToIntegerThrowOnInfinity** → **TruncateToInteger**
> * **ToIntegerWithoutRounding** → **InsistInteger**
> * **ToPositiveInteger** → **TruncateToPositiveInteger**

Reply:
> ah, i wouldn’t expect ToPositiveInteger to throw on zero.

I guess "positive" was used because in ECMA-262 "non-negative" is typically used when zero is included, whereas "positive" is (often) used when zero isn't included.

Reply:
I was also surprised, but not because zero is out of range—my surprise was because "To…" operations generally don't throw at all except when conversion is deemed impossible at a type-to-type level or to propagate exceptions from ECMAScript code, neither of which apply to ToPositiveInteger(0). Why should the behavior diverge from similarly-named operations, e.g. ToIntegerOrInfinity("foo") or ToIntegerOrInfinity(3.14) or ToLength(Infinity) or ToIndex(-1)?

That's why I want to make the names _dissimilar_ and more indicative of behavior.

Reply:
Since zeros have a sign in JS, I assumed +0 was included in "positive integers".